src/main/scala/org/broadinstitute/clio/webservice/headers.scala

```diff
@@ -0,0 +1,40 @@
+package org.broadinstitute.clio.webservice
+
+/*
+ * These traits just implement Akka-HTTP's header rendering methods.
+ * A ModeledCustomHeader implementation will probably want to extend
+ * one of these.  RequestResponseHeader is the only one used now for
+ * the OidcHeader classes.  See {@link OidcAccessToken} for example.
+ */
+
+/**
+  * Headers that render in neither requests nor responses.
+  */
+trait InternalHeader {
+  def renderInRequests():Boolean = false
+  def renderInResponses():Boolean = false
+}
+
+/**
+  * Headers that render only in requests.
+  */
+trait RequestHeader {
+  def renderInRequests():Boolean = true
+  def renderInResponses():Boolean = false
+}
+
+/**
+  * Headers that render only in responses.
+  */
+trait ResponseHeader {
+  def renderInRequests():Boolean = false
```

Comment:
For each of these return types, add a space between `:` and `Boolean`. Can also be done by selecting all `⌘A` and then selecting Code > Reformat Code `⌥⌘L`.